web/core/components/views/views-list.tsx

```diff
@@ -25,7 +25,7 @@ export const ProjectViewsList = observer(() => {
 
   if (loader || !projectViews || !filteredProjectViews) return <ViewListLoader />;
 
-  if (filteredProjectViews.length === 0 && projectViews) {
+  if (filteredProjectViews.length === 0 && projectViews.length > 0) {
```

Comment:
I think the issue meant to change the empty view to a different one. Check with Pranjal.